data/actionsmap/yunohost.yml

```diff
@@ -1351,6 +1368,21 @@ tools:
                     help: Show private data (domain, IP)
                     action: store_true
 
+        ### tools_urlavailable()
+        urlavailable:
```

Comment:
I am for the use of "yunohost tools url-available" . 
Dash is used everywhere but in app cmd. We should fix this and correct app cmd (there is depreciation mechanism in moulinette)